integration_tests/e2e/changeNonResidentialRoomType/index.cy.ts

```diff
@@ -0,0 +1,151 @@
+import AuthSignInPage from '../../pages/authSignIn'
+import Page from '../../pages/page'
+import LocationFactory from '../../../server/testutils/factories/location'
+import NonResidentialRoomPage from '../../pages/nonResidentialRoom'
+import NonResidentialRoomTypeChangePage from '../../pages/nonResidentialRoom/setNonResidentialChangeType'
+
+context('View non-residential rooms', () => {
```

Comment:
The title of this context should probably be something like "Change non-residential room type"